src_assets/common/assets/web/Navbar.vue

```diff
@@ -1,60 +1,93 @@
-                    </li>
-                </ul>
-            </div>
-        </div>
-    </nav>
+  <nav class="navbar navbar-light navbar-expand-lg navbar-background header">
+    <div class="container-fluid">
+      <a class="navbar-brand" href="/" title="Sunshine">
+        <img src="/images/logo-sunshine-45.png" height="45" alt="Sunshine">
+      </a>
+      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent"
+              aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+          <li class="nav-item">
+            <a class="nav-link" href="/"><i class="fas fa-fw fa-home"></i> {{ $t('navbar.home') }}</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="/pin"><i class="fas fa-fw fa-unlock"></i> {{ $t('navbar.pin') }}</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="/apps"><i class="fas fa-fw fa-stream"></i> {{ $t('navbar.applications') }}</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="/config"><i class="fas fa-fw fa-cog"></i> {{ $t('navbar.configuration') }}</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="/password"><i class="fas fa-fw fa-user-shield"></i> {{
+                $t('navbar.password')
+              }}</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="/troubleshooting"><i class="fas fa-fw fa-info"></i> {{
+                $t('navbar.troubleshoot')
+              }}</a>
+          </li>
+          <li class="nav-item">
+            <ThemeToggle/>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
 </template>
 
 <script>
+import ThemeToggle from './ThemeToggle.vue'
+
 export default {
-    created() {
-        console.log("Header mounted!")
-    },
-    mounted() {
-        let el = document.querySelector("a[href='" + document.location.pathname + "']");
-        if (el) el.classList.add("active")
-        let discordWidget = document.createElement('script')
-        discordWidget.setAttribute('src', 'https://app.lizardbyte.dev/js/discord.js')
-        document.head.appendChild(discordWidget)
-    }
+  components: { ThemeToggle },
+  created() {
+    console.log("Header mounted!")
+  },
+  mounted() {
+    let el = document.querySelector("a[href='" + document.location.pathname + "']");
+    if (el) el.classList.add("active")
+    let discordWidget = document.createElement('script')
+    discordWidget.setAttribute('src', 'https://app.lizardbyte.dev/js/discord.js')
+    document.head.appendChild(discordWidget)
+  }
 }
 </script>
 
 <style>
-.nav-link.active {
-    font-weight: 500;
+.navbar-background {
```

Comment:
Would it make sense to separate out all the styles into the sunshine.css file?

Reply:
I'd rather have each component with it's own style, with time I think it makes things easier, but I was thinking how could I move all the default colors, text sizes and stuff to sunshine.css, kind of creating a design system or theme , but in another PR